content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring Virtualization and Apps in TrueNAS SCALE"
+weight: 50
+tags:
+- scalevm
+- scaleapps
+- scalenetwork
+- scalevlan
+- scalelagg
+- scaledatashare
+---
+
+{{< toc >}}
+
+Configuring TrueNAS SCALE to work with virtualized features, such as VMs and Apps, is a part of the setup process that when optimized takes advantage of the network storage capabilities that SCALE offers. 
+
+## Before You Begin
+
+This article assumes that you have the latest release version of TrueNAS SCALE installed on your system. The following steps comprise a list of configuration prerequisites that you have completed and are familiar with before beginning VM and App installation.
+
+### Network Prerequisites
+
+The primary network interface is configured as part of the SCALE installation process. Go to **Network > Global Configuration** screen in the TrueNAS web UI to determine that the default gateway, host name, domain and DNS name servers have been configured correctly. See [Console Setup Menu Configuration]({{< relref "ConsoleSetupMenuScale.md" >}}) for more information on network settings.
+
+### Optional: Network VLAN 
```

Comment:
This is not parallel construction for the headers. You can consider changing all header to something like this: 
### Network Configuration
### VLAN Configuration
### Storage Configuration
### Shares Configuration

Reply:
Headers have been changed.